### PR TITLE
In readerController allow page numbers of more than 2 digits. fix #27

### LINKED ISSUE #27: Show more than 2 digits in num pages in reader
Actually, maxSize of pageNumField and totalPagesField is set to 50.
When the number of pages exceeds 2 digits, only 2 digits are shown fully.
In pageNumField it is possible to insert cursor and scroll but in totalPagesField it is not possible since the field is disabled.

So instead of using maxWidth I propose to use the number of colums (pref number of columns set to 2 in fxml and a binding added in the controller to allow the fields' number of columns to grow and shrink following the number of digits).

I'll send a PR to fix this.
Attached is a screenshot showing the phenomenon (the left textfield contains the number 128 but only 12 is shown, the right box already has my fix applied and the full 128 is shown).
Screenshot_20191101_144442

### PR DESCRIPTION
**Pull request by bayang**
Allows the textFields with number of pages in reader controller to grow and shrink according to the number of digits, thus showing all the digits.

### CODE REVIEW

**Comment by bayang** [comment]
Actually commit 43381ed was planned to come in a separate PR, my bad.

File: src/main/java/com/faltro/houdoku/controller/ReaderController.java [hunk 1]
```diff
@@ -137,6 +138,19 @@ public void initialize() {
...
+        totalPagesField.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> ob, String o, String n) {
+                totalPagesField.setPrefColumnCount(totalPagesField.getText().length());
+            }
+        });
+        pageNumField.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> ob, String o, String n) {
+                pageNumField.setPrefColumnCount(pageNumField.getText().length() +1);
```
Comment (xgi) [inline src/main/java/com/faltro/houdoku/controller/ReaderController.java]: I don't think this +1 is supposed to be here?

Comment (bayang) [reply src/main/java/com/faltro/houdoku/controller/ReaderController.java]: No, you're right, it is not supposed to be here. 33acaf3 should fix this.
Thanks !

**Comment by xgi** [comment]
Perfect, thanks!

**Pull request merged**